homeworks/08-functional-reactive-programming/src/utils.ts

```diff
@@ -0,0 +1,97 @@
+function randomIntFromInterval(min: number, max: number): number {
```

Comment:
This file contains functions with different responsibilities. Will be fine to split it.